releasenotes/notes/cws-7-54-bb2551a1d930853b.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    CWS: Add support ``write`` event on Windows.
+  - |
+    CWS: Add support for ``rename`` event on Windows.
```

```suggestion
    CWS: Add support for ``delete`` events on Windows.
```
we already have a rename release note, but we need one for delete